src/lib/components/navbar/Navbar.component.js

```diff
@@ -120,6 +148,13 @@ function NavBar({
           <ProductNameSpan>{productName}</ProductNameSpan>
         </NavbarMenuItem>
       </NavbarMenu>
+      <NavbarTabs>
+        {tabs.length > 0 &&
```

Comment:
you can remove the `tab.length > 0` check here, since the default value is an empty array you can just leave the `.map()`.